### ISSUE TITLE
All Contributors should respect a .prettierrc if one is found

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [all-contributors/cli#347]
**Describe the bug**
I have a `.prettierrc` in a repo that indicates its `.allcontributorsrc` file should be formatted as JSON with tabs. But the allcontributors bot is still formatting it as JSON with two spaces.

See https://github.com/JoshuaKGoldberg/template-typescript-node-package/pull/99/files#diff-cb22095514d0cab1f755ae3a7902f7d64d10199da210fbd1eb2b29eb758b38d2

**To Reproduce**
1. Create a new repo
2. Create a `.pretterrc` file like https://github.com/JoshuaKGoldberg/template-typescript-node-package/blob/3b1bbc2b57a4ec40492d1e94814d3e48d9deb910/.prettierrc
3. Set up allcontributors

**Expected behavior**
Prettier settings should be applied when formatting the file.

**Comment by Berkmann18** [comment]
That's an excellent point, do you know of any project or have a PoC that respects `.prettierrc`?

**Comment by JoshuaKGoldberg** [comment]
Yup, the repo linked in the OP uses Prettier for formatting files. It has an explicit override to ignore `.all-contributorsrc` because of this issue. 

https://github.com/JoshuaKGoldberg/template-typescript-node-package/blob/129e4560bb966339ec5bf4b73741e07fb5b5262f/.prettierignore#L4-L5

**Comment by Berkmann18** [comment]
That makes sense! Do you have an idea of how the bot could appropriately take it into account?

**Comment by JoshuaKGoldberg** [comment]
In theory you can run `npx prettier .all-contributorsrc --write` and that'll pick up whatever Prettier config is checked into the repo (as long as the bot has done at least a shallow clone).

**Comment by Berkmann18** [comment]
I see, so it seems like something that could be integrated into the CLI (especially since the bot uses it to an extent).
